Pocket/pocket-ios
# feat(premium): offline premium upsell

## Summary
* Offline premium upsell

## References 
* [IN-1207](https://getpocket.atlassian.net/browse/IN-1207)

## Implementation Details
* Added offline premium upsell screen

## Test Steps
* as an offline user, go to settings
* click "Go Premium"
* should see normal upsell screen
* go back to settings
* go offline
* click "Go Premium"
* should see modified upsell with bottom notification

## PR Checklist:

## Screenshots
<img width="395" alt="image" src="https://user-images.githubusercontent.com/111293209/224833089-d772d481-0e5b-4b0e-8867-5f91d89fc1f9.png">


[IN-1207]: https://getpocket.atlassian.net/browse/IN-1207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
